tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -1,433 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The Gluon vision data pipeline use case seems still relevant. Why is the test removed?